### ISSUE TITLE
Bug: Duplication. `.. Linter | Spacings | Tabsize` already defined in `Obsidian | Settings | Options | Editor | tab size`

### ISSUE THREAD

**Issue by businessBoris** [platers/obsidian-linter#1150]
[I love the plugin. Please don't take this as criticism.] 

- [X] I have verified that I am on the latest version of the Linter

## Describe the Bug

Obsidian collects tabs-to-spaces figure which shouldn't be incongruous to the linters spaces-to-tabs figure. Linter provides no default, which should be collected from the system tabs-to-spaces figure.

## How to Reproduce

Steps to reproduce the behaviour:

1. Go to `Obsidian | Settings | Options | Editor | tab size` ("Number of spaces a tab is equivalent to")
2. Observe slider indicating figure.
3. Go to  `Third-party plugin | Linter | Spacings | Tabsize` ("Number of spaces that will be converted to a tab")
4. Observe no default.
5. Observe that this figure is unneeded and is already provided in the host application.  It doesn't make sense that the tabs-to-spaces and spaces-to-tabs would be different.

**Comment by rsenna** [comment]
Good point. I also like this plugin **a lot** (great job @pjkaufman), but it _does_ introduce unnecessary complexity sometimes… E.g. too much explanation, excessive settings and so on.

In this case the setting **already exists** as part of Obsidian, therefore it should not redefine it at all.
